tests/systemtests/cli.go

```diff
@@ -217,7 +217,7 @@ func (c CLIWrapper) runWithInput(args []string, input io.Reader) (output string,
 		cmd := exec.Command(locateExecutable(c.execBinary), args...) //nolint:gosec // test code only
 		cmd.Dir = WorkDir
 		cmd.Stdin = input
-		return cmd.CombinedOutput()
+		return cmd.Output()
```

Comment:
there is a lot of noise by the simdv2 in the output, therefore It is ignored here but added to the error msg in cmd by Go stdlib